.azure-pipelines/templates/test.yml

```
@@ -12,6 +12,7 @@ jobs:
     - job: test_${{ replace(replace(replace(replace(job.test, '/', '_'), '.', '_'), '-', '_'), '@', '_') }}
       displayName: ${{ job.name }}
       container: $[ variables.defaultContainer ]
+      timeoutInMinutes: 90  # 85 + 5
```

Since ansible-test implements its own timeout of 60 minutes, this could be set lower:

```suggestion
      timeoutInMinutes: 65
```